src/Documents.jl

```diff
@@ -113,19 +113,23 @@ end
 
 struct ContentsNode
     pages       :: Vector{String} # Which pages should be included in contents? Set by user.
+    mindepth    :: Int            # Minimum header level that should be displayed. Set by user.
     depth       :: Int            # Down to which level should headers be displayed? Set by user.
     build       :: String         # Same as for `IndexNode`s.
     source      :: String         # Same as for `IndexNode`s.
     elements    :: Vector         # (order, page, anchor)-tuple for constructing links.
 
     function ContentsNode(;
             Pages  = [],
-            Depth  = 2,
+            Depth  = 1:2,
             build  = error("missing value for `build` in `ContentsNode`."),
             source = error("missing value for `source` in `ContentsNode`."),
             others...
         )
-        new(Pages, Depth, build, source, [])
+        if Depth isa Integer
+            Depth = 1:Depth
+        end
+        new(Pages, Depth.start, Depth.stop, build, source, [])
```

Comment:
I don't think the `.start` and `.stop` fields are technically documented?

```suggestion
        new(Pages, first(Depth), last(Depth), build, source, [])
```